go/vt/vtorc/inst/analysis_dao.go

```diff
@@ -267,6 +268,10 @@ func GetReplicationAnalysis(keyspace string, shard string, hints *ReplicationAna
 		JOIN vitess_keyspace ON (
 			vitess_tablet.keyspace = vitess_keyspace.keyspace
 		)
+		JOIN vitess_shard ON (
+			vitess_tablet.keyspace = vitess_shard.keyspace
+			AND vitess_tablet.shard = vitess_shard.shard
+		)
```

Comment:
The fact we use `JOIN` rather then `LEFT JOIN` suggests `vitess_shard` is always populated? Just clarifying the records have to always exist.

Reply:
The `vitess_shard` should be populated. If it isn't then I think it is better VTOrc not run any failover/fix at all until it is populated. That is why the JOIN. So, if the record doesn't exist, the tablet also doesn't show up in the result.